server/server_suite_test.go

```diff
@@ -0,0 +1,14 @@
+package server_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Avoid using `.` Not a good practice in case the two libraries have the same function names

Reply:
Noted 👍 . But these imports are generated automatically when initializing repositories to have ginkgo tests.